libraries/botbuilder-dialogs-adaptive/src/entityInfo.ts

```diff
@@ -83,6 +83,9 @@ export interface NormalizedEntityInfos {
     [name: string]: Partial<EntityInfo>[];
 }
 
+/**
+ * Extended information about an entity including $instance data.
```

Comment:
Is this ok? $instance

Reply:
I just copied it from the line [10](https://github.com/southworks/botbuilder-js/pull/172/files/6d532cc55bc5bcbe691a39ad1341709f8bf5467a#diff-ce80c52b5d143b729099e5997b0ddc12L10) of this file, I checked out in dotnet and it's the same comment but nothing to reference to `$instance` as well.

Reply:
Ok, let's keep it then. Thanks.